### PR TITLE
feat: ✨ surface infrastructure as code reference links [ROAD-573]

### PR DESCRIPTION
**Pull request by bastiandoetsch**
image

### CODE REVIEW

File: src/main/kotlin/snyk/iac/IacSuggestionDescriptionPanel.kt [hunk 1]
```diff
@@ -272,6 +276,27 @@
         return remediationPanel
     }
```
Comment (ArtsiomCh) [inline src/main/kotlin/snyk/iac/IacSuggestionDescriptionPanel.kt:277]: minor: why can't we use `issue.references.size` (and adjust indexes below)?

File: src/main/kotlin/snyk/iac/IacSuggestionDescriptionPanel.kt [hunk 2]
```diff
@@ -272,6 +276,27 @@
...
+                referenceRowCount + 1,
+                1,
+                Insets(20, 0, 20, 0),
+                50,
+                -1
+            )
+
+            panel.add(boldLabel("References"), baseGridConstraints(row = 1))
+            issue.references.forEachIndexed { index, s ->
+                val label = labelProvider.createLinkLabel(URL(s), s)
```
Comment (ArtsiomCh) [inline src/main/kotlin/snyk/iac/IacSuggestionDescriptionPanel.kt:293]: minor: not obvious why `row=7` and what `this` is here. Shell we move that addition to main panel init{} as it done for others panels (like remediationPanelWithTitle)?

**Review by ArtsiomCh — APPROVED** [review]
LGTM
some minor comments could be addressed.

File: src/main/kotlin/snyk/iac/IacSuggestionDescriptionPanel.kt [hunk 3]
```diff
@@ -272,6 +276,27 @@ class IacSuggestionDescriptionPanel(
         return remediationPanel
     }
 
+    private fun addIssueReferences() {
+        val referenceRowCount = if (issue.references.isNotEmpty()) issue.references.size + 1 else 0
```
Comment (ArtsiomCh) [inline src/main/kotlin/snyk/iac/IacSuggestionDescriptionPanel.kt:280]: minor: just use `issue.references.size` ?

File: src/main/kotlin/snyk/iac/IacSuggestionDescriptionPanel.kt [hunk 4]
```diff
@@ -272,6 +276,27 @@ class IacSuggestionDescriptionPanel(
...
+                50,
+                -1
+            )
+
+            panel.add(boldLabel("References"), baseGridConstraints(row = 1))
+            issue.references.forEachIndexed { index, s ->
+                val label = labelProvider.createLinkLabel(URL(s), s)
+                panel.add(label, baseGridConstraints(2 + index))
+            }
+            this.add(panel, panelGridConstraints(row = 7))
```
Comment (ArtsiomCh) [inline src/main/kotlin/snyk/iac/IacSuggestionDescriptionPanel.kt:296]: minor: not obvious why `row=7` and what `this` is here. Shell we move that addition to main panel init{} as it done for others panels (like remediationPanelWithTitle)?

Comment (bastiandoetsch) [reply src/main/kotlin/snyk/iac/IacSuggestionDescriptionPanel.kt:296]: Sounds good!

**Pull request merged**